utilities/node_activator/lib/node_activator.ex

```diff
@@ -10,23 +10,25 @@ defmodule NodeActivator do
   @doc """
   Activates Node.
   """
-  @spec run(binary()) :: {:ok, node() | pid()} | {:error, term()}
-  def run(name) do
-    if Node.alive?() do
-      {:ok, Node.self()}
-    else
-      case :os.type() do
-        {:unix, _} -> launch_epmd(port: @epmd_port)
-        {:win32, _} -> launch_epmd(port: @epmd_port, daemon: false)
-      end
+  @spec run(binary()) :: {:ok, node()}
```

Comment:
The typespec of `Node.start/3` is `@spec start(node(), :longnames | :shortnames, non_neg_integer()) ::
  {:ok, pid()} | {:error, term()}`, so `NodeActivator.run/0` may return `{:error, term()}`.